test/unit/webapps/test_request_scoped_sqlalchemy_sessions.py

```diff
@@ -0,0 +1,162 @@
+import asyncio
```

Comment:
The intention here is for ``galaxy-webapps`` to depend on Python 3.7 or newer? I so, we should fine a way to encode that in its setup.py maybe?

Reply:
I think galaxy-webapps should work fine on Python 3.6, as long as you don't build an ASGI app. I don't mind putting a python 3.7 restriction in setup.py, but it's not 100% necessary.